### ISSUE TITLE
Names array contains tokens

### ISSUE THREAD

**Issue by mr-jonix** [mtgjson/mtgjson#112]
names array of cards that generate tokens includes names of these tokens, while it shouldn't.

Example
```
{
            "artist": "Victor Adame Minguez",
            "borderColor": "black",
            "colorIdentity": [
                "U"
            ],
            "colors": [
                "U"
            ],
            "convertedManaCost": 5.0,
            "foreignData": [
                {
                    "language": "German",
                    "multiverseId": 435534,
                    "name": "Zauberschwindel",
                    "text": "Neutralisiere einen Zauberspruch deiner Wahl. Erzeuge X farblose Schatz-Artefaktspielsteine, wobei X gleich den umgewandelten Manakosten des Zauberspruchs ist. Sie haben „{T}, opfere dieses Artefakt: Erhöhe deinen Manavorrat um ein Mana einer beliebigen Farbe.\"",
                    "type": "Spontanzauber"
                },
                {
                    "language": "Spanish",
                    "multiverseId": 437627,
                    "name": "Rechazo próspero",
                    "text": "Contrarresta el hechizo objetivo. Crea X fichas de artefacto Tesoro incoloras, donde X es el coste de maná convertido de ese hechizo. Tienen \"{T}, sacrificar este artefacto: Agrega un maná de cualquier color a tu reserva de maná\".",
                    "type": "Instantáneo"
                },
                {
                    "language": "French",
                    "multiverseId": 435833,
                    "name": "Flouerie de sort",
                    "text": "Contrecarrez le sort ciblé. Créez X jetons d'artefact incolore Trésor, X étant le coût converti de mana de ce sort. Ils ont « {T}, sacrifiez cet artefact : Ajoutez un mana de la couleur de votre choix à votre réserve. »",
                    "type": "Éphémère"
                },
                {
                    "language": "Italian",
                    "multiverseId": 436132,
                    "name": "Raggiro Magico",
                    "text": "Neutralizza una magia bersaglio. Crea X pedine artefatto Tesoro incolori, dove X è il costo di mana convertito di quella magia. Hanno \"{T}, Sacrifica questo artefatto: Aggiungi un mana di un qualsiasi colore alla tua riserva di mana\".",
                    "type": "Istantaneo"
                },
                {
                    "language": "Japanese",
                    "multiverseId": 436431,
                    "name": "呪文詐欺",
                    "text": "呪文１つを対象とし、それを打ち消す。無色の宝物・アーティファクト・トークンをＸ個生成する。Ｘはその呪文の点数で見たマナ・コストに等しい。それらは「{T}, このアーティファクトを生け贄に捧げる：あなたのマナ・プールに好きな色１色のマナ１点を加える。」を持つ。",
                    "type": "インスタント"
                },
                {
                    "language": "Korean",
                    "multiverseId": 436730,
                    "name": "주문 사취",
                    "text": "주문을 목표로 정한다. 그 주문을 무효화한다. 무색 보물 마법물체 토큰 X개를 만든다. X는 그 주문의 전환마나비용이다. 그 토큰들은 \"{T}, 이 마법물체를 희생한다: 당신의 마나풀에 원하는 색의 마나 한 개를 담는다.\"를 가진다.",
                    "type": "순간마법"
                },
                {
                    "language": "Portuguese (Brazil)",
                    "multiverseId": 437029,
                    "name": "Logro Mágico",
                    "text": "Anule a mágica alvo. Crie X fichas de artefato incolor do tipo Tesouro, sendo X o custo de mana convertido daquela mágica. Elas têm \"{T}, sacrifique este artefato: Adicione um mana de qualquer cor à sua reserva de mana.\"",
                    "type": "Mágica Instantânea"
                },
                {
                    "language": "Russian",
                    "multiverseId": 437328,
                    "name": "Стянуть Заклинание",
                    "text": "Отмените целевое заклинание. Создайте X фишек бесцветного артефакта Клад, где X — конвертированная мана-стоимость того заклинания. У них есть способность «{T}, пожертвуйте этот артефакт: добавьте одну ману любого цвета в ваше хранилище маны».",
                    "type": "Мгновенное заклинание"
                },
                {
                    "language": "Chinese Simplified",
                    "multiverseId": 437926,
                    "name": "咒语欺诈",
                    "text": "反击目标咒语。派出X个无色珍宝衍生神器，X为该咒语的总法术力费用。它们具有「{T}，牺牲此神器：加一点任意颜色的法术力到你的法术力池中。」",
                    "type": "瞬间"
                },
                {
                    "language": "Chinese Traditional",
                    "multiverseId": 438225,
                    "name": "咒語欺詐",
                    "text": "反擊目標咒語。派出X個無色珍寶衍生神器，X為該咒語的總魔法力費用。它們具有「{T}，犧牲此神器：加一點任意顏色的魔法力到你的魔法力池中。」",
                    "type": "瞬間"
                }
            ],
            "frameVersion": "2015",
            "hasFoil": true,
            "hasNonFoil": true,
            "isReserved": false,
            "layout": "normal",
            "legalities": {
                "1v1": "Legal",
                "brawl": "Legal",
                "commander": "Legal",
                "duel": "Legal",
                "frontier": "Legal",
                "future": "Legal",
                "legacy": "Legal",
                "modern": "Legal",
                "penny": "Legal",
                "standard": "Legal",
                "vintage": "Legal"
            },
            "manaCost": "{3}{U}{U}",
            "multiverseId": 435235,
            "name": "Spell Swindle",
            "names": [
                "Spell Swindle",
                "Treasure"
            ],
            "number": "82",
            "originalText": "Counter target spell. Create X colorless Treasure artifact tokens, where X is that spell's converted mana cost. They have \"{T}, Sacrifice this artifact: Add one mana of any color to your mana pool.\"",
            "originalType": "Instant",
            "printings": [
                "PXLN",
                "XLN"
            ],
            "rarity": "rare",
            "rulings": [
                {
                    "date": "2017-09-29",
                    "text": "For spells with {X} in their mana costs, use the value chosen for X to determine the spell’s converted mana cost."
                },
                {
                    "date": "2017-09-29",
                    "text": "You may target a spell that can’t be countered. When Spell Swindle resolves, the target spell will be unaffected, but you’ll still get Treasures."
                }
            ],
            "subtypes": [],
            "supertypes": [],
            "text": "Counter target spell. Create X colorless Treasure artifact tokens, where X is that spell's converted mana cost. They have \"{T}, Sacrifice this artifact: Add one mana of any color.\"",
            "type": "Instant",
            "types": [
                "Instant"
            ],
            "uuid": "6e619ada-e9ce-4758-afd8-8def853877eb"
        }
```

**Comment by mr-jonix** [comment]
Also, DFC cards that produce tokens have the face names concatenated.

Example, Westvale Abbey

```
"name": "Ormendahl, Profane Prince",
            "names": [
                "Westvale Abbey // Ormendahl, Profane Prince",
                "Human Cleric"
            ]
```

**Comment by ZeldaZach** [comment]
I'm pretty sure you're using 4.0.2, which this is a bug on. Just tested with my fix and it seems good to go. Thanks for the report!

**Comment by KingSupernova31** [comment]
This has not been fixed, see #119.
